gtk3-macros/src/lib.rs

```diff
@@ -21,38 +21,57 @@ use syn::{parse_macro_input, DeriveInput};
 /// Specify that `MyWidget` is using a composite template and load the
 /// template file the `composite_template.ui` file.
 ///
-/// ```compile_fail
-/// use gtk::prelude::*;
-/// use gtk::CompositeTemplate;
-///
-/// #[derive(Debug, Default, CompositeTemplate)]
-/// #[template(file = "composite_template.ui")]
-/// struct MyWidget {
-///     #[template_child]
-///     pub label: TemplateChild<gtk::Label>,
-/// }
-/// ```
-///
 /// Then, in the `ObjectSubclass` implementation you will need to call
 /// `bind_template` in the `class_init` function, and `init_template` in
 /// `instance_init` function.
 ///
+/// ```no_run
+/// use gtk::prelude::*;
+/// use gtk::glib;
+/// use gtk::subclass::prelude::*;
 ///
-/// ```compile_fail
-/// #[glib::object_subclass]
-/// impl ObjectSubclass for MyWidget {
-///        const NAME: &'static str = "MyWidget";
-///        type Type = super::MyWidget;
-///        type ParentType = gtk::Widget;
-///
-///        fn class_init(klass: &mut Self::Class) {
-///            Self::bind_template(klass);
-///        }
-///
-///        fn instance_init(obj: &glib::subclass::InitializingObject<Self>) {
-///            obj.init_template();
-///        }
-///    }
+/// mod imp {
+///     use super::*;
+/// 
+///     #[derive(Debug, Default, CompositeTemplate)]
```

Comment:
```suggestion
///     #[derive(Debug, Default, gtk::CompositeTemplate)]
```

So that this is probably fixed

```
 error: cannot find derive macro `CompositeTemplate` in this scope
  --> src/lib.rs:36:30
   |
10 |     #[derive(Debug, Default, CompositeTemplate)]
   |                              ^^^^^^^^^^^^^^^^^
```